frontend/src/metabase/ui/components/navigation/NavLink/NavLink.styled.tsx

```diff
@@ -26,14 +26,14 @@ export const getNavLinkOverrides = (): MantineThemeOverride["components"] => ({
             },
 
             backgroundColor: theme.fn.themeColor("brand"),
-            color: "white",
+            color: theme.fn.themeColor("text-white"),
```

Comment:
since this is a mantine override settings. I assume the color name would work right out of the gate e.g.
```suggestion
            color: "text-white",
```

Can you double check if this is working as intended?

Reply:
I think for consistency I like the idea of using the `theme.fn.themeColor` for both of them. But yeah let's check if that still works